File: jugwang/week1/medium.py
def solution(clothes):
    dic = {}
    
    answer = 1
    
    for _, k in clothes:
        if k in dic:
            dic[k] += 1
        else:
            dic[k] = 1
        
    for v in dic.values():
        answer *= (v+1)
    
    return answer-1

File: juha/week1/medium.js
function solution(clothes) {
  let closet = {};
  for (let item of clothes) {
    let name = item[0];
    let type = item[1];

    if (closet[type]) {
      closet[type] += 1;
    } else {
      closet[type] = 1;
    }
  }
  let answer = 1;
  for (let type in closet) {
    answer = answer * (closet[type] + 1);
  }
  return answer - 1;
}
